verification/policy.go

```diff
@@ -135,10 +135,10 @@ func validateTrustStore(statement TrustPolicy) error {
 
 // ValidatePolicyDocument validates a policy document according to it's version's rule set.
 // if any rule is violated, returns an error
-func ValidatePolicyDocument(policyDoc *PolicyDocument) error {
+func (policyDoc *PolicyDocument) ValidatePolicyDocument() error {
 	// Constants
 	supportedPolicyVersions := []string{"1.0"}
-	supportedVerificationPresets := []string{"strict", "permissive", "audit", "skip"}
+	supportedVerificationLevels := []string{"strict", "permissive", "audit", "skip"}
```

Comment:
Are you going to replaces this with levels define in https://github.com/notaryproject/notation-go/pull/55 ?

Reply:
Yes. I have a TODO to do that.